test-result-summary-client/src/Build/Summary/ResultGrid.jsx

```diff
@@ -44,6 +44,18 @@ class Cell extends Component {
                                             <a href={groups[group].buildUrl} target="_blank" rel="noopener noreferrer">Jenkins Link</a>
                                         </div>
                                     );
+                                } else if (group === 'perf'){
+                                    element = (
+                                        <div>
+                                            Test Target: {target} <br />
+                                            Build Result: {result} <br />
+                                            Result Summary: {Object.keys(groups[group].testSummary).map((key) => {
+                                                return <div key={key}>{key}: {groups[group].testSummary[key]}</div>;
+                                            })} 
+                                            Build Result is from parser, (not CI build) <br />
+                                            <a href={groups[group].buildUrl} target="_blank" rel="noopener noreferrer">Jenkins Link</a>
+                                        </div>
+                                    );
```

Comment:
It is better to not duplicate the code. Otherwise, we need to handle additional branch if we make changes in the future.

In the original else block, add 
```
{group === 'perf' ? "Build Result is from TestBenchmarkParser (not CI build)" : ""}
```